From 59f0b28341ee780b76503c51cb25220152b42c64 Mon Sep 17 00:00:00 2001
From: WangYuli <wangyuli@uniontech.com>
Date: Mon, 16 Dec 2024 10:23:06 +0800
Subject: [PATCH 303/360] deepin: arm64: config: Update arm64 desktop config

Follow changes:
 *Update configs by "make savedefconfig"
 *Change pstore_ram from y to m
 *Disable zboot

Reported-by: Jie Fan <fanjie@uniontech.com>
Suggested-by: Wentao Guan <guanwentao@uniontech.com>
Signed-off-by: WangYuli <wangyuli@uniontech.com>
---
 arch/arm64/configs/deepin_arm64_desktop_defconfig | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/configs/deepin_arm64_desktop_defconfig b/arch/arm64/configs/deepin_arm64_desktop_defconfig
index 1c2f2c184733..d51c375f5955 100644
--- a/arch/arm64/configs/deepin_arm64_desktop_defconfig
+++ b/arch/arm64/configs/deepin_arm64_desktop_defconfig
@@ -206,7 +206,6 @@ CONFIG_ZSWAP=y
 CONFIG_ZSWAP_COMPRESSOR_DEFAULT_ZSTD=y
 CONFIG_ZSWAP_ZPOOL_DEFAULT_ZSMALLOC=y
 CONFIG_ZBUD=y
-CONFIG_Z3FOLD=m
 CONFIG_ZSMALLOC_STAT=y
 CONFIG_SLAB_FREELIST_RANDOM=y
 CONFIG_SLAB_FREELIST_HARDENED=y
@@ -821,7 +820,6 @@ CONFIG_FW_CFG_SYSFS_CMDLINE=y
 CONFIG_ARM_FFA_TRANSPORT=m
 CONFIG_EFI_VARS_PSTORE=m
 CONFIG_EFI_VARS_PSTORE_DEFAULT_DISABLE=y
-CONFIG_EFI_ZBOOT=y
 CONFIG_EFI_BOOTLOADER_CONTROL=m
 CONFIG_EFI_CAPSULE_LOADER=m
 CONFIG_RESET_ATTACK_MITIGATION=y
@@ -1376,6 +1374,11 @@ CONFIG_I40E_DCB=y
 CONFIG_I40EVF=m
 CONFIG_ICE=m
 CONFIG_FM10K=m
+CONFIG_IGC=m
+CONFIG_MGBE=m
+CONFIG_MGBEVF=m
+CONFIG_MXGBE=m
+CONFIG_MXGBEVF=m
 CONFIG_JME=m
 CONFIG_MVMDIO=m
 CONFIG_SKGE=m
@@ -3963,7 +3966,7 @@ CONFIG_ROMFS_FS=m
 CONFIG_ROMFS_BACKED_BY_BOTH=y
 CONFIG_PSTORE_CONSOLE=y
 CONFIG_PSTORE_PMSG=y
-CONFIG_PSTORE_RAM=y
+CONFIG_PSTORE_RAM=m
 CONFIG_UFS_FS=m
 CONFIG_UFS_FS_WRITE=y
 CONFIG_EROFS_FS=m
-- 
2.20.1

